templates/insights/blocks/quote.html

```diff
@@ -2,22 +2,17 @@
 
 <div class="quote">
     <div class="container">
-        <img class="quote__icon" src="{% static 'images/fontawesome-svgs/quote-left.svg' %}" alt="" />
-        {% if value.heading %}
-            <{{ heading_level }} class="sr-only">{{ value.heading }}</{{ heading_level }}>
-        {% endif %}
-
         {% if value.attribution %}
-            <figure>
-                <blockquote class="quote__container quote__text">
+            <figure class="quote__container">
+                <blockquote class="quote__text">
                     {{ value.quote }}
                 </blockquote>
                 <figcaption class="quote__citation">
                     <cite>{{ value.attribution }}</cite>
                 </figcaption>
             </figure>
         {% else %}
-            <blockquote class="quote__container quote__text">
+            <blockquote class="quote__text">
```

Comment:
Double-checking this... is it okay that there is no `quote_container` when there is no attribution value?

Reply:
My mistake, that should be on the parent element. Fixed now, thanks for spotting it!